### PR TITLE
Tidy up changelogs

### PR DESCRIPTION
**Pull request by Vexed01**
### Type

- [x] Bugfix
- [ ] Enhancement
- [ ] New feature

### Description of the changes
Self-explanatory, I've used the following "rules":
```
All entries have full stops.
Commands are ``[p]command``, and there is always an empty line.

```
Some may say that bullet point lists don't have full stops, I'm willing to change this if that is the case.

I'll try to keep this update this once a week as stuff is merged, you may want to put off merging this until closer to 3.2 release.

### CODE REVIEW

File: changelog.d/downloader/2902.bugfix.rst [hunk 1]
```diff
@@ -1 +1 @@
-findcog no longer attempts to find a cog for commands without one.
\ No newline at end of file
+``findcog`` no longer attempts to find a cog for commands without one.
```
Comment (Drapersniper) [inline changelog.d/downloader/2902.bugfix.rst]: Missing [p]

File: changelog.d/mod/2924.fix.rst [hunk 2]
```diff
@@ -1 +1 @@
-``modset deletedelay`` doesn't delete non-commands.
\ No newline at end of file
+``modset deletedelay`` doesn't delete non-commands.
```
Comment (Drapersniper) [inline changelog.d/mod/2924.fix.rst]: Missing [p]

**Review by Drapersniper — COMMENTED** [review]
Looking good, all for consistency here!

**Comment by mikeshardmind** [comment]
Not really convinced this is the best way to handle this.

At the end of a cycle, we'd run `towncrier build` which assembles this all into a single file. This would then get added as a doc page, probably with editing done then for ease of consistency and adding additional formatting / links / additional release notes.

Leaving this open for now.

**Comment by Vexed01** [comment]
See https://github.com/Cog-Creators/Red-DiscordBot/pull/3091#issuecomment-570160913.

**Pull request closed without merging**